keyboards/sneakbox/lilbae/info.json

```
@@ -0,0 +1,69 @@
+{
+    "manufacturer": "Sneakbox",
+    "maintainer": "mujimanic",
+    "bootloader": "caterina",
```

```suggestion
    "bootloader": "atmel-dfu",
```
My hardware uses atmel-dfu and this was the bootloader specified in your `rules.mk` as well so this seems to be in error. If your newer batch PCB's are in fact using caterina bootloader there should be separate revision subdirectories created.